src/BootstrapModalManager.tsx

```diff
@@ -10,21 +10,28 @@ const Selector = {
 };
 
 export default class BootstrapModalManager extends ModalManager {
-  adjustAndStore = (prop, element, adjust) => {
+  private adjustAndStore<T extends keyof CSSStyleDeclaration>(
+    prop: T,
+    element: HTMLElement,
+    adjust: number,
+  ) {
     const actual = element.style[prop];
     element.dataset[prop] = actual;
     css(element, {
-      [prop]: `${parseFloat(css(element, prop)) + adjust}px`,
+      [prop]: `${parseFloat(css(element, prop as any)) + adjust}px`,
```

Comment:
the `T` isn't doing a much here if we have to cast to any right below yeah?

Reply:
It kind of is... `T` allows us to read `element.style` safely, but `css(...)` uses a different typing, so... maybe some sort of intersection type between `keyof CSSStyleDeclaration` and `keyof CSS.Properties`?